ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -1,49 +1,135 @@
+                class="yes-radio"
+                label="Yes"
+                active-class="selected-radio"
+                :value="true"
+                data-test-id="yes-ownership-radiobtn"
+              />
+              <v-radio
+                id="no-option"
+                class="no-radio"
+                label="No"
+                active-class="selected-radio"
+                :value="false"
+                data-test-id="no-ownership-radiobtn"
+              />
+            </v-radio-group>
+          </v-col>
+        </v-row>
+        <v-row v-if="isOwnLand">
+          <v-col cols="9" offset="3">
+            <v-divider class="mx-0 divider-mt" />
+            <p class="mb-n2 paragraph-mt" data-test-id="yes-paragraph">
+              <b>Note:</b> Land ownership or registered lease of the land for 3 years or more
+              must be verifiable through the BC Land Title and Survey Authority (LTSA)
+              or other authorized land authority.
+            </p>
+          </v-col>
+        </v-row>
+        <v-row v-if="!isOwnLand && isOwnLand!=null">
+          <v-col cols="9" offset="3">
+            <v-divider class="mx-0 divider-mt" />
+            <p class="mb-n2 paragraph-mt" data-test-id="no-paragraph">
+              <b>Note:</b> Written permission and tenancy agreements from the landowner
+              may be required for the home to remain on the land.
+              <br><br>
+              Relocation of the home onto land that the homeowner does not own or hold a
+              registered lease of 3 years or more may require additional permits from
+              authorities such as the applicable Municipality, Regional District, First
+              Nation, or Provincial Crown Land Office.
+            </p>
+          </v-col>
+        </v-row>
+      </v-row>
+    </v-form>
   </v-card>
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive, toRefs, watch } from 'vue-demi'
+import { computed, defineComponent, reactive, ref, toRefs, watch } from 'vue-demi'
 import { useStore } from '@/store/store'
 import { storeToRefs } from 'pinia'
+import { useMhrValidations } from '@/composables'
+import { FormIF } from '@/interfaces'
 
 export default defineComponent({
   name: 'HomeLandOwnership',
-  setup () {
+  components: {},
+  props: {
+    validate: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup (props) {
     const {
       setMhrRegistrationOwnLand
     } = useStore()
+    const { getMhrRegistrationValidationModel } = storeToRefs(useStore())
+    const {
+      MhrCompVal,
+      MhrSectVal,
+      setValidation
+    } = useMhrValidations(toRefs(getMhrRegistrationValidationModel.value))
     const {
       getMhrRegistrationOwnLand
     } = storeToRefs(useStore())
+    const leaseOrOwnForm = ref(null) as FormIF
 
     const localState = reactive({
-      isOwnLand: !!getMhrRegistrationOwnLand.value
+      isOwnLand: null || getMhrRegistrationOwnLand.value,
```

Comment:
Not sure about this logic here, it will always get the value from the store.

Reply:
This line of code was 'isOwnLand: getMhrRegistrationOwnLand.value || false', I thought there might be a specific reason for listing out the default value of isOwnLand. I have delete the unecessary part.